src/python/WMCore/WMSpec/WMWorkloadTools.py

```diff
@@ -215,7 +215,7 @@ def validateSiteLists(arguments):
         whiteList = [whiteList]
     if not isinstance(blackList, list):
         blackList = [blackList]
-    res = (set(whiteList) & set(blackList))
+    res = [s for s in list(set(whiteList) & set(blackList)) if s]
```

Comment:
IMO, fixing 215/217 makes it more obvious what you are protecting against, but I'm not asking you to change it.